code/lib/cli-storybook/src/doctor/getDuplicatedDepsWarnings.ts

```diff
@@ -32,7 +32,6 @@ export const disallowList = [
   Object.keys(rendererPackages),
   Object.keys(frameworkPackages),
   'storybook',
-  '@storybook/instrumenter',
```

Comment:
are we going to ban projects that use @storybook/instrumenter or @storybook/testing-library etc? otherwise it's still useful to check for duplicated dependencies